engine/engine.go

```diff
@@ -230,5 +230,47 @@ func cmd() *cobra.Command {
 		},
 	})
 
+	cmd.AddCommand(&cobra.Command{
+		Use:   "sign-tls-cert [public-key-file] [OPTIONAL output-file]",
+		Short: "Creates and signs a certificate for usage as TLS client certificate",
+		Long: "Creates a CSR for the given public-key using a TLS client certificate template. " +
+			"It's signed with the vendor CA. " +
+			"The certificate is printed to stdout in PEM encoded form and optionally stored in the given location",
```

Comment:
not "and optionally stored", it's printed to stdout or stored at the given location.